tests/Magick.NET.Tests/Statistics/PerceptualHashTests/TheSumSquaredDistanceMethod.cs

```diff
@@ -1,15 +1,38 @@
 ﻿// Copyright Dirk Lemstra https://github.com/dlemstra/Magick.NET.
 // Licensed under the Apache License, Version 2.0.
 
+using System;
 using ImageMagick;
 using Xunit;
 
 namespace Magick.NET.Tests;
 
 public partial class PerceptualHashTests
 {
+    public class TestPerceptualHash : IPerceptualHash
+    {
+        public IChannelPerceptualHash GetChannel(PixelChannel channel)
+        {
+            return null;
+        }
+
+        public double SumSquaredDistance(IPerceptualHash other)
+        {
+            throw new System.NotImplementedException();
+        }
+    }
+
     public class TheSumSquaredDistanceMethod
     {
+        [Fact]
+        public void ShouldThrowNotSupportedExceptionIfCustomImplementationDoesNotHaveExpectedChannels()
+        {
+            using var image = new MagickImage(Files.ImageMagickJPG);
+            var phash = image.PerceptualHash();
+
+            Assert.Throws<NotSupportedException>(() => phash.SumSquaredDistance(new TestPerceptualHash()));
```

Comment:
You can also check the message of the exception like we do in other places.

Reply:
Added.